packages/gamut/src/BetaSticker/index.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+export const BetaSticker: React.FC = () => {
+  return (
+    <svg
+      width="54"
+      height="28"
+      viewBox="0 0 54 28"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <g filter="url(#filter0_d)">
+        <rect x="2" width="52" height="26" fill="#FCF0E6" />
+      </g>
+      <path
+        d="M8 18.6H11.792C14.256 18.6 15.936 17.592 15.936 15C15.936 13.592 15.232 12.792 14.288 12.376C15.056 11.848 15.536 10.984 15.536 9.992C15.536 7.576 13.84 7 11.6 7H8V18.6ZM10.4 11.768V8.952H11.568C12.544 8.952 13.024 9.432 13.024 10.36C13.024 11.288 12.544 11.768 11.568 11.768H10.4ZM13.392 15.08C13.392 16.104 12.88 16.616 11.696 16.616H10.4V13.528H11.696C12.88 13.528 13.392 14.04 13.392 15.08Z"
+        fill="#10162F"
+      />
+      <path
+        d="M18.5677 18.6H25.5917V16.536H20.9677V13.704H25.2717V11.64H20.9677V9.064H25.5917V7H18.5677V18.6Z"
+        fill="#10162F"
+      />
+      <path
+        d="M31.1675 18.6H33.5675V9.064H36.8475V7H27.8875V9.064H31.1675V18.6Z"
+        fill="#10162F"
```

Comment:
For monochrome svgs we like to make sure that all fill / stroke colors use [currentColor](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#currentcolor_keyword) which will make the color dynamic to the parent :D 

```suggestion
        fill="currentColor"
```

You can test this out locally in storybook by using `<Text color="hyper" />` in the docs page.